src/main/java/com/mark0wka/entity/Attribute.java

```diff
@@ -0,0 +1,33 @@
+package com.mark0wka.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "object_attribute_list")
+@Data
+@NoArgsConstructor
+public class Attribute {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "attr_id")
+    private int attrId;
```

Comment:
attr_id shouldn't be marked as generated value
it must be the same for all objects for a particular attribute